quarkus/config-api/src/main/java/org/keycloak/config/CachingOptions.java

```diff
@@ -82,4 +90,41 @@ public enum Stack {
             .buildTime(true)
             .build();
 
+    public static final Option<String> CACHE_REMOTE_HOST = new OptionBuilder<>(CACHE_REMOTE_HOST_PROPERTY, String.class)
+            .category(OptionCategory.CACHE)
+            .description(String.format("The hostname of the remote server for the remote store configuration. "
+                    + "It replaces the 'host' attribute of 'remote-server' tag of the configuration specified via XML file (see '%s' option.). "
+                    + "If the option is specified, '%s' and '%s' are required as well and the related configuration in XML file should not be present.", 
+                    CACHE_CONFIG_FILE_PROPERTY, CACHE_REMOTE_USERNAME_PROPERTY, CACHE_REMOTE_PASSWORD_PROPERTY))
+            .buildTime(true)
```

Comment:
None of the new options must be a build time option, as all build time options are baked into the image, and would require an image rebuild to change. We never want to store passwords in the image, and we never want to rebuild the image when we change the host.

Reply:
Thank you for pointing it out. Updated.